packages/twenty-front/src/modules/settings/integrations/components/SettingsIntegrationRemoteTableSyncStatusToggle.tsx

```diff
@@ -4,24 +4,26 @@ import { Toggle } from '@/ui/input/components/Toggle';
 import { RemoteTableStatus } from '~/generated-metadata/graphql';
 
 export const SettingsIntegrationRemoteTableSyncStatusToggle = ({
-  table,
+  tableName,
+  tableStatus,
   onSyncUpdate,
 }: {
-  table: { id: string; name: string; status: RemoteTableStatus };
+  tableName: string;
+  tableStatus: RemoteTableStatus;
   onSyncUpdate: (value: boolean, tableName: string) => Promise<void>;
 }) => {
   const [isToggleLoading, setIsToggleLoading] = useState(false);
 
   const onChange = async (newValue: boolean) => {
     if (isToggleLoading) return;
     setIsToggleLoading(true);
-    await onSyncUpdate(newValue, table.name);
+    await onSyncUpdate(newValue, tableName);
     setIsToggleLoading(false);
```

Comment:
@thomtrp for the record setIsToggleLoading(false) will never be executed if there is a sync error (for instance, the table has no id). We could wrap onSyncUpdate in a try and execute setIsToggleLoading(false) in a finally block. But I noticed that then its making the situation worse because the toggle stays on because that's how the implementation of Toggle works.
I will look into this in a further PR has this problem already exists.